src/Microsoft.Azure.WebJobs.Host/Blobs/Listeners/BlobTriggerExecutor.cs

```diff
@@ -110,7 +128,16 @@ internal class BlobTriggerExecutor : ITriggerExecutor<IStorageBlob>
                     BlobName = value.Name,
                     ETag = possibleETag
                 };
-                await _queueWriter.EnqueueAsync(message, cancellationToken);
+                // BlobTriggerMessage to JObject
+                // we can save two functionid, not a big deal
+                if (_queueWriter == null)
```

Comment:
I don't understand this conditional - I assume this is just temporary?

Reply:
TODO change the unit test